src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -104,6 +102,14 @@ abstract contract DelegationManagerStorage is IDelegationManager {
     /// @dev Salt is used in the `registerOperatorToAVS` function.
     mapping(address => mapping(bytes32 => bool)) public operatorSaltIsSpent;
 
+    /**
+     * @notice Minimum delay enforced by this contract per Strategy for completing queued withdrawals. Measured in blocks, and adjustable by this contract's owner,
+     * up to a maximum of `MAX_WITHDRAWAL_DELAY_BLOCKS`. Minimum value is 0 (i.e. no delay enforced).
+     * @dev Note that the withdrawal delay is not enforced on withdrawals of 'beaconChainETH', as the EigenPods have their own separate delay mechanic
+     * and we want to avoid stacking multiple enforced delays onto a single withdrawal.
```

Comment:
I don't think this is true -- are we changing this?

Native ETH currently does use the withdrawal delay.

Reply:
> EigenPods have their own separate delay mechanic

They do, mainly for partial withdrawals, which don't need to go through the DMgr queue anyway. 

For a full withdrawal, which does go through the queue, the completed withdrawal is sent directly (not via the delayed withdrawal router)

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/files#diff-d323417ded59a593f4ea410225a9167b24198c4d2c05cbc0fe9af47d34b88810R640
Added back min global delay